castle/cms/tasks/files.py

```diff
@@ -87,10 +87,10 @@ def youtube_video_edited(obj):
 
         # we can only edit if file has NOT changed
         # otherwise, it will be reuploaded and original deleted
-        if md5 is not None:
+        if sha256 is not None:
```

Comment:
will this change mean that ALL files would need to be reprocessed?

we may need to keep md5 here (with the 'usedforsecurity=False' param) or figure out a migration path that doesn't involve reprocessing all files?

Reply:
I reimplement md5 and added a `useforsecurity` attribute